package.json

```diff
@@ -58,7 +58,7 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "Azure Resource Manager Tools Configuration Settings",
+      "title": "ARM Tools Configuration Settings",
```

Comment:
Where does this string get displayed?

Reply:
In the settings dialog:

![image](https://user-images.githubusercontent.com/6913354/63474093-b5403580-c42c-11e9-8647-4993897ba8df.png)

Reply:
Okay lol but `Tools Configuration Settings` is clearly the problem here - not `Azure Resource Manager`

Reply:
True on that one, looks like it actually never gets displayed.